sirepo_bluesky/sirepo_detector.py

```diff
@@ -168,9 +208,15 @@ def describe(self):
 
     def unstage(self):
         super().unstage()
-        self._resource_id = None
+        self._resource_document = None
         self._result.clear()
 
+    def collect_asset_docs(self):
+        items = list(self._asset_docs_cache)
+        self._asset_docs_cache.clear()
+        for item in items:
+            yield item
```

Comment:
Interesting that it works with non-flyers, and it's good it's done the same way.

Reply:
That is by design!